docs/source/system_reference_guide/s3-inventory.ipynb

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
<p>Maybe one line about what this notebook is doing would be helpful in the future? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>